packages/admin-ui-codemod/src/__tests__/space-tokens-review.js

```diff
@@ -0,0 +1,270 @@
+const defineInlineTest = require('jscodeshift/dist/testUtils').defineInlineTest
```

Comment:
I really like these tests, @anitavincent ! They're pretty straight forward to read and have good coverage! 👏